susemanager-utils/testing/docker/scripts/test_cobbler.sh

```diff
@@ -25,4 +25,25 @@ cobblerd
 # execute the tests
 
 cd /usr/share/cobbler/tests
-pytest --junitxml=/reports/cobbler.xml
+
+pytest --junitxml=/reports/cobbler.xml \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_anon_bind_negative[True-test-bad] \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_anon_bind_positive[True] \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_anon_bind_positive[True-test-test] \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_cadir_positive[/etc/ssl/certs-/etc/ssl/ldap.crt-/etc/ssl/ldap.key] \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_cafile_positive[/etc/ssl/ca-slapd.crt-/etc/ssl/ldap.crt-/etc/ssl/ldap.key] \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_ldaps_positive[/etc/ssl/ca-slapd.crt-/etc/ssl/ldap.crt-/etc/ssl/ldap.key] \
+    --deselect=modules/authentication/ldap_test.py::TestLdap::test_user_bind_positive[False-uid=user,dc=example,dc=com-test-test-test] \
+    --deselect=modules/authentication/pam_test.py::TestPam::test_authenticate \
+    --deselect=tftpgen_test.py::test_copy_single_distro_file \
+    --deselect=utils_test.py::test_blender \
```

Comment:
This is the single most important test for us. Don't skip that.

Reply:
Thanks for briging the attention about this particular one.

In case that we exclude this one, it should be only temporary until we really fix it in the Cobbler package and adjust the test expectation there.

Reply:
I've updated the list of deselected tests, now this important test is not excluded.